test/services/ReviewServiceSpec.scala

```diff
@@ -128,32 +129,107 @@ class ReviewServiceSpec extends UnitSpec with MockFactory with ReviewData with A
   }
 
   "Calling the changeStatus method" when {
-    "the ID identifies a valid process" should {
-      "return to indicate the process status was updated in the database" in new Test {
+    "the ID identifies a valid process" when {
+      "the status is submitted for 2i review" should {
+        "indicate the process status was updated in the database" in new Test {
 
-        val expected: RequestOutcome[Unit] = Right(())
+          val expected: RequestOutcome[Unit] = Right(())
 
-        MockApprovalRepository
-          .changeStatus("validId", statusChangeInfo)
-          .returns(Future.successful(expected))
+          MockApprovalRepository
+            .getById("validId")
+            .returns(Future.successful(Right(approvalProcess)))
 
-        whenReady(service.changeStatus("validId", statusChangeInfo)) { result =>
-          result shouldBe expected
+          MockApprovalRepository
+            .changeStatus("validId", statusChangeInfo)
+            .returns(Future.successful(expected))
+
+          whenReady(service.changeStatus("validId", StatusSubmittedFor2iReview, statusChangeInfo)) { result =>
+            result shouldBe expected
+          }
+        }
+      }
+
+      "the status is published" should {
```

Comment:
Change from submitted for 2i review to published

Reply:
changed description